classes/pricing/aws_base_pricing_client.php

```diff
@@ -77,7 +63,7 @@ class aws_ets_pricing_client {
     /**
      * The ServiceCode for Amazon Elastic Transcode Services.
      */
-    const SERVICE_CODE = 'AmazonETS';
+    const SERVICE_CODE = '';
 
     /**
      * Map of AWS region codes to location names used by \Aws\Pricing\PricingClient.
```

Comment:
might be out of scope but can we grab this list of local_aws instead of here, or is this list a smaller subset of regions?

Reply:
This should likely be a helper in local_aws, but currently it only provides the admin setting to control it, not a data list of regions. https://github.com/catalyst/moodle-local_aws/issues/30